FAQ.md

```diff
@@ -0,0 +1,153 @@
+# Development
```

Comment:
Should we maybe call this file `CONTRIBUTING.md` instead? Would be more standard

Reply:
Indeed!